### DISCUSSION TITLE
The icon is too big.

### DISCUSSION THREAD

**Discussion by earthmoon** [xiaoyifang/goldendict-ng#764] [Q&A]
Hello,

When translate with Goldendict, the triangle icon is too big. 
![image](https://github.com/xiaoyifang/goldendict-ng/assets/45897471/20654aed-6ff0-4c76-8c1c-122c65f49ccc)

Thanks.

**Comment by xiaoyifang** [comment]
Do you use some custom stylesheet?

**Reply by earthmoon** [reply]
> Do you use some custom stylesheet?


No, I haven't.

**Reply by xiaoyifang** [reply]
click right mouse on the icon and open inspect ,the devtool will show you which css class is taking effect

**Reply by earthmoon** [reply]
> click right mouse on the icon and open inspect ,the devtool will show you which css class is taking effect

May I clicked the wrong place, as I can't find the devtool.
![image](https://github.com/xiaoyifang/goldendict-ng/assets/45897471/c91d0a5d-f438-49e8-8adf-e929651b19a2)

**Reply by xiaoyifang** [reply]
then click at the nearby of this icon

**Reply by earthmoon** [reply]
> nearby

The same. Please check the video from [the link](https://bthc-my.sharepoint.com/:v:/g/personal/mr_iem_im/ERDjYLMkbUJGjFexKSLrvpgBw7pqwF0Eh6NwgdtJ7vB9IA?e=W7kcZL).

**Reply by xiaoyifang** [reply]
press F12 to open the devtool ,then navigate to the icon.

**Reply by xiaoyifang** [reply]
or this place
![image](https://github.com/xiaoyifang/goldendict-ng/assets/105986/5b9684f9-13fa-49a0-bb70-6ff46efd8c48)

**Reply by earthmoon** [reply]
> or this place ![image](https://user-images.githubusercontent.com/105986/242134048-5b9684f9-13fa-49a0-bb70-6ff46efd8c48.png)

The same. Is this one:
><p>(c) 2008-2013 Konstantin Isakov. Licensed under GPLv3 or later.</p>

Please check this [link](https://bthc-my.sharepoint.com/:v:/g/personal/mr_iem_im/EVEPhniEW2FNkVUJuS-P30oBaKuCHQfG57TdoauDVPZGmQ?e=vz8i9z).

**Reply by xiaoyifang** [reply]
I got it ,the devtool does not exist in the popup windows.

How does it display in mainwindows

**Reply by earthmoon** [reply]
> I got it ,the devtool does not exist in the popup windows.
> 
> How does it display in mainwindows

I guess nothing changed in the main window. Please check the [video](https://bthc-my.sharepoint.com/:v:/g/personal/mr_iem_im/EQJgaroQCd1FqFp2mEz-lM0ByCgw3GneOaBVKCCtnoYFfQ?e=HKeFhi).

**Reply by xiaoyifang** [reply]
use mainwindow's devtool to debug this issue.

**Reply by earthmoon** [reply]
> debug

I can't understand what I should do. May you make an example for me? Thanks.

**Reply by xiaoyifang** [reply]
https://github.com/xiaoyifang/goldendict-ng/assets/105986/637b87dc-1c69-43f0-a4f7-25454180320f

**Reply by earthmoon** [reply]
> vXwUm1sIXt.mp4

Is this one? 
`<img class="gdcollapseicon" id="expandicon-b88a9403c8bb9a944c34f7f93224cc83" title="Collapse article">`

**Reply by xiaoyifang** [reply]
yes . that is.

check whether the css style has been affected by dictionary's stylesheet

**Reply by earthmoon** [reply]
> yes . that is.
> 
> check whether the css style has been affected by dictionary's stylesheet

That is too difficult for me. Thanks.

**Comment by xiaoyifang** [comment]
or just choose a different style
![image](https://github.com/xiaoyifang/goldendict-ng/assets/105986/c9e9e501-c42a-4a2d-b0c3-b4576dd2dee0)

**Reply by earthmoon** [reply]
> or just choose a different style ![image](https://user-images.githubusercontent.com/105986/242139865-c9e9e501-c42a-4a2d-b0c3-b4576dd2dee0.png)

Thanks. This is a another way.

**Reply by earthmoon** [reply]
Hello,

This problem disappears if I use the version of 6.4.3, instead of  5.15.2.  I am confused why you released the 5.15.2 and 6.4.3 at the same time? And even for Windows there are five versions about 6.4.3. What's the reason?

Thanks.

**Reply by xiaoyifang** [reply]
as to qt6.4.3 Windows version ,only has two .
![image](https://github.com/xiaoyifang/goldendict-ng/assets/105986/565b0f94-fb55-47c3-b8c3-002342102861)

to suit various requirement.

**Reply by earthmoon** [reply]
> as to qt6.4.3 Windows version ,only has two . ![image](https://user-images.githubusercontent.com/105986/242149863-565b0f94-fb55-47c3-b8c3-002342102861.png)
> 
> to suit various requirement

I am very confused of these:

1. As you have updated the version to 6.4.3, why did you update the version of 5.15.2? Most people always update the lastest one, not others;
2. Form [the link](https://github.com/xiaoyifang/goldendict-ng/releases/tag/v23.05.01-WuYi.230501.6c8b0c1c), tthere are five files about Windows, which all refer to 0501. They are: `6.4.3-GoldenDict-ng-v23.05.01-WuYi-Installer.exe`, ` 6.4.3-GoldenDict-ng-v23.05.01-WuYi-xapian-Installer.exe`,  `6.4.3-GoldenDict.exe_windows-2019_20230501-xapian.zip`, `6.4.3-GoldenDict.exe_windows-2019_20230501.zip` and `6.4.3-GoldenDict.exe_windows-2022_20230501.zip`. What's the difference among them and why did you release so many?

Thanks.

**Reply by xiaoyifang** [reply]
some of the listed versions have already been remove .  check the latest version. 
https://github.com/xiaoyifang/goldendict-ng/releases

**Reply by earthmoon** [reply]
> some of the listed versions have already been remove . check the latest version. https://github.com/xiaoyifang/goldendict-ng/releases

Thanks. But the lastest one even contain any versions for Windows.

**Reply by xiaoyifang** [reply]
https://github.com/xiaoyifang/goldendict-ng/discussions/187

**Reply by earthmoon** [reply]
> #187

Thanks.
